backend/Models/Request/ConfirmOrRejectSightingRequest.cs

```diff
@@ -0,0 +1,16 @@
+using System.ComponentModel.DataAnnotations;
+using System;
+
+namespace WhaleSpotting.Models.Request
+{
+    public class ConfirmOrRejectRequest
+    {
+        [Required]
+        public int SightingId { get; set; }
+
+        // true == approved
+        // false == rejected
+        [Required]
+        public bool isApproved { get; set; }
```

Comment:
I think instead, have this be the desired confirmation status, then you can avoid any ambiguity:
```suggestion
        [Required]
        public ConfirmationStatus NewConfirmationStatus { get; set; }
```